test/unittest/xmlparser/CMakeLists.txt

```diff
@@ -173,7 +184,7 @@ if(NOT ((MSVC OR MSVC_IDE) AND EPROSIMA_INSTALLER))
             ${PROJECT_SOURCE_DIR}/include
             ${PROJECT_BINARY_DIR}/include
             )
-            
+
```

Comment:
As you will change it in unitest/log/CMakeLists.txt, change all `target_link_libraries` to cmake generator expressions. See the main src/cpp/CMakeLists.txt to see and example.